cabal-install/src/Distribution/Client/HttpUtils.hs

```diff
@@ -500,7 +500,7 @@ wgetTransport prog =
             warningMsg     =  "the 'wget' transport currently doesn't support"
                            ++ " range requests, which wastes network bandwidth."
                            ++ " To fix this, set 'http-transport' to 'curl' or"
-                           ++ " 'plain-http' in '~/.cabal/config'."
+                           ++ " 'plain-http' in '~/.config/cabal/config'."
```

Comment:
Again, maybe better to use $CABAL_DIR here, because for some people it will be ~/.cabal still…